guides-and-tutorials/best-practices-to-snapshot-the-chainstate.md

```diff
@@ -0,0 +1,210 @@
+# Best Practices for Stacks Chainstate Snapshots
+
+{% hint style="info" %}
+**Intended audience**: Solo Stackers, Stacking pool operators, and node operators who need to create reliable chainstate backups.
+{% endhint %}
+
+Regular snapshots of your Stacks chainstate help you recover quickly when things go wrong. This guide shows you how to create and manage chainstate snapshots properly.
+
+{% hint style="warning" %}
+**Critical**: Always shut down your Stacks node properly before creating a snapshot. Creating snapshots while the node is running will result in corrupted state data.
+{% endhint %}
+
+## Shutdown Procedure
+
+To produce a valid chainstate backup, the node should be stopped gracefully before making a copy. The following steps will correctly shutdown the Stacks node:
+
+1. **Check node status** before shutdown
+   ```bash
+   # Verify if the node is responsive
+   curl http://localhost:20443/v2/info
+   ```
+
+2. **Initiate graceful shutdown**
+   - For Docker: `docker stop stacks-node` (allows at least 10 seconds for graceful shutdown)
+   - For systemd: `systemctl stop stacks-node`
+   - For manual processes: Send SIGTERM signal
```

Comment:
I think we should add specifics here, ex:
```
kill $(ps aux | grep stacks-node | grep -v grep | awk '{print $2}')
```
or even a small shell scipt/commands